util/app-config/Cargo.toml

```diff
@@ -12,7 +12,7 @@ serde_plain = "0.3.0"
 toml = "0.5"
 path-clean = "0.1.0"
 ckb-logger = { path = "../../util/logger" }
-ckb-logger-service = { path = "../../util/logger-service" }
+ckb-logger-config = { path = "../../util/logger-config" }
 sentry = "0.16.0"
```

Comment:
useless dependencies?

Reply:
No.

Useless dependencies will be found by [`devtools/ci/check-cargotoml.sh`](https://github.com/nervosnetwork/ckb/blob/057b7e2/devtools/ci/check-cargotoml.sh) and CI will be failed.